platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/SwirldsPlatform.java

```diff
@@ -656,6 +675,9 @@ public class SwirldsPlatform implements Platform {
                 intakeEventCounter,
                 platformWiring.getKeystoneEventSequenceNumberOutput());
 
+        final LongSupplier intakeQueueSizeSupplier =
+                oldStyleIntakeQueue != null ? oldStyleIntakeQueue::size : platformWiring.getIntakeQueueSizeSupplier();
```

Comment:
It's always better to avoid negatives if you can:
`final LongSupplier intakeQueueSizeSupplier =
                oldStyleIntakeQueue == null ? platformWiring.getIntakeQueueSizeSupplier() : oldStyleIntakeQueue::size;`